verilog/analysis/verilog_linter_test.cc

```diff
@@ -439,6 +441,87 @@ TEST(VerilogLinterDocumentationTest, AllRulesMarkdown) {
   EXPECT_TRUE(absl::StrContains(stream.str(), "Enabled by default:"));
 }
 
+TEST(VerilogLinterDocumentationTest, PrintLintRuleFile) {
+  auto config_status = verilog::LinterConfigurationFromFlags("");
+  EXPECT_TRUE(config_status.ok());
+  if (!config_status.ok()) {
+    return;
+  }
+  const LinterConfiguration &config = *config_status;
+
+  // Generate Line Rule File
+  std::ostringstream stream;
+  verilog::GetLintRuleFile(&stream, config);
+
+  // Spot-check a few patterns, must mostly make sure generation
+  // works without any fatal errors.
+  // NOTE: This will break if/when the rules change so this part
+  // of the test is not ideal.
+  EXPECT_TRUE(absl::StrContains(stream.str(), "always-comb"));
```

Comment:
In tests, we can make use of the `testing::HasSubstr` for slightly more readable way of writing this (grep for this pattern in other tests):

```c++
EXPECT_THAT(stream.str(), HasSubstr("always-comb"));
```

Reply:
I think I did it this way as there are a number of other tests in that file using the EXPECT_TRUE(sbsl::StrContains... pattern. Happy to change if you prefer.

Reply:
Similar here: let's do cleanup of existing improvement places to a separate PR.